openapi.yaml

```diff
@@ -3492,12 +3502,12 @@ paths:
                     type: string
                     nullable: true
                     description: |-
-                      Name of the plan the user has subscribed to.
+                      Name of the plan the user has subscribed to, if any.
 
-                      Overrides the default plan of the back-end, but back-ends
-                      MAY also allow overriding this plan for each individual
-                      processing request (e.g. job or service) with the
-                      corresponding `plan` property.
+                      Back-ends resolve billing plans in the following order:
+                      1. If provided, use the `plan` in the processing request.
+                      2. If provided, use the plan exposed here.
+                      3. Otherwise, use the `default_plan` exposed through `GET /`.
```

Comment:
I'm not sure it's useful to explain this priority scheme here. I think it complicates the description unnecessarily. It's discussed in a more appropriate place already. 

What could be useful to mention is whether a user can be subscribed to multiple plans (I think not, if I understand correctly). 
Another subtlety is whether there is a difference between a missing "default_plan" or a "null" default plan (e.g. backend does not support plans vs user has no plan)

Reply:
* Agreed, removed the priority scheme.
* Clarified that only a single plan is supported.
* null: I think I can actually remove the nullable as it has the same meaning as "not present".